torch/nn/quantized/modules/linear.py

```diff
@@ -142,6 +146,36 @@ def extra_repr(self):
             self.in_features, self.out_features, self.scale, self.zero_point, self.weight().qscheme()
         )
 
+    def __repr__(self):
+        # We don't want to show `LinearPackedParams` children, hence custom
```

Comment:
Is this the right approach to take? I think we should show users that there is a LinearPackedParams class and it details instead of hiding it. Particularly with torchbind, there should be no reason to hide this module in the repr

Reply:
I think it is right -- there is no need for the user to know these details. The repr is usually used just to see the architecture. Showing the `LinearPackedParam` is not very useful. At the same time, if we unpack and show it, it will just cluttern the hole console